src/themes/DarkTheme.js

```diff
@@ -1,16 +1,40 @@
+import { lightenDarkenColor } from './helpers';
+import { convertToRgb } from './helpers';
+
 const colors = {
```

Comment:
Consider changing colors names according to http://chir.ag/projects/name-that-color